client/tso_dispatcher.go

```diff
@@ -138,12 +139,27 @@ func (c *tsoClient) updateTSODispatcher() {
 	})
 }
 
-type deadline struct {
-	timer  <-chan time.Time
+// TSDeadline is used to watch the deadline of each tso request.
+type TSDeadline struct {
```

Comment:
Why change this struct to be exported? Since it's an internal struct, I prefer to keep it not exported.